web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/Elements.kt

```diff
@@ -60,7 +60,12 @@ import org.w3c.dom.HTMLVideoElement
 import org.w3c.dom.Text
 import org.w3c.dom.css.CSSStyleSheet
 
-typealias AttrBuilderContext<T> = AttrsBuilder<T>.() -> Unit
+@Deprecated(
+    message = "Renamed to AttrsBuilder<T>",
+    replaceWith = ReplaceWith("AttrsBuilder<T>", "org.jetbrains.compose.web.dom.AttrsBuilder")
+)
+typealias AttrBuilderContext<T> = AttrsBuilder<T>
+typealias AttrsBuilder<T> = AttrsScope<T>.() -> Unit
```

Comment:
I think I see your idea is to have consistent naming: AttrsBuilder and ContentBuilder are both lambdas with corresponding context.

at the same time, shouldn't `typealias AttrsBuilder<T>` be an alias for a new AttrsScope (or even AttrsScopeBuilder)? so it still can be instantiated/used in cases when someone already uses old AttrsBuilder directly in their code? 

And if yes, then it would need to be in `org.jetbrains.compose.web.attributes` package to keep it compatible with existing codebases (in external projects). And then typealias AttrsBuilder can be marked Depreacted. 

The idea is to make a change as less breaking as possible.

Reply:
Good point, I'll revisit this.

Reply:
Ok after reviewing this I think nothing needs to change.
* Old `typealias AttrBuilderContext<T> = AttrsBuilder<T>.() -> Unit` was in this file, so it needs to stay here deprecated to maintain comatability
* New `AttrsBuilder` references the `AttrsScope` interface for consistency with `ContentBuilder` leaving `AttrsScopeBuilder` implementation hidden as an implementation detail